content/python/concepts/matplotlib/terms/legend/legend.md

```diff
@@ -0,0 +1,88 @@
+---
+Title: '.legend()'
+Description: 'Places a legend on the axes and access to manipulate parameters of the legend.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`pyplot.legend()`** function returns a legend on the axes of a plot or subplot.
+
+## Syntax
+
+```pseudo
+plot_instance.legend(handles, labels)
+```
+
+The `.legend()` accepts zero or more parameters and can be called on a valid `plot_instance` of the `pyplot` module. When called with no parameters, the `.legend()` function will attempt to add the elements automatically. The following parameters can be used in the `.legend()` function:
+
+|  Parameter Name  | Data Type(s)         | Usage                                                                                                  |
+| :--------------: | -------------------- | ------------------------------------------------------------------------------------------------------ |
+|    `handles`     | sequence of `Artist` | `handles` is utilized if more control is desired with each entry on the plot and how they are labeled. |
+|     `labels`     | list of strings       | `labels` is a list of strings that will display the labels in the order they are entered.              |
+|      `loc`       | int, string          | `loc` is used to determine the position of the legend on the axes. Defaults to `best`.                 |
+|      `ncol`      | int                  | `ncol` is used to determine how many columns the legend will have. Defaults to 1.                      |
+|    `fontsize`    | int, string          | `fontsize` is used change the font size the labels. Defaults to 11.                                    |
+|   `labelcolor`   | list, string, `None` | `labelcolor` is used to change the color of the label(s). Defaults to `None`                           |
+|     `title`      | string, `None`       | `title` is used to insert a title for the legend. Defaults to `None`.                                  |
+| `title_fontsize` | int, string, `None`  | `title_fontsize` is used to change the font size of the legend `title`. Defaults to `None`.            |
+
+## Example
+
+In the following example, the `.legend()` method will be shown using the iris data set which is available using seaborn or pandas. In the first example the `handles` parameter will be demonstrated along with using seaborn to import the dataset. In the second example `legend()` will be demonstrated without using the `handles` parameter along with using pandas to import the dataset:
```

Comment:
```suggestion
In the following example, the `.legend()` method will be shown using the iris data set which is available using `seaborn` or [`pandas`](https://www.codecademy.com/resources/docs/pandas). In the first example, the `handles` parameter will be demonstrated, using `seaborn` to import the dataset. In the second example, `legend()` will be demonstrated without using the `handles` parameter along, using pandas to import the dataset:
```

Seaborn can be added as a link after the `seaboarn` entry is added!

Reply:
@cjunhyuk is there a reason we want two examples? If so, can we include the reasoning here as to why we would use `seaborn` over `pandas` or vice versa?

Reply:
The first example is a base case with no alterations to any of the parameters for the legend. The second example is demonstrating some of the parameters and showing it. 
The use of `pandas` compared to `seaborn` is negligible in this instance. Since I was showing two examples I just showed two different ways to accessing the same built in dataset.
We can use only 1 example if that would be preferred.